python/src/equistore/operations/reduce_over_samples.py

```diff
@@ -321,89 +397,162 @@ def sum_over_samples(
+    """
+
+    return _reduce_over_samples_block(
+        block=block, sample_names=sample_names, reduction="std"
+    )
+
+
+def std_over_samples(tensor: TensorMap, sample_names: List[str]) -> TensorMap:
+    r"""Compute the standard deviation of a :py:class:`TensorMap`, combining the samples
+    according to ``sample_names``.
 
     This function creates a new :py:class:`TensorMap` with the same keys as
     as the input ``tensor``, and each :py:class:`TensorBlock` is obtained
     performing the std deviation of the corresponding input :py:class:`TensorBlock`
-    over the ``samples_names`` indices.
+    over the ``sample_names`` indices.
 
-    ``samples_names`` indicates over which dimensions in the samples the mean is
+    ``sample_names`` indicates over which dimensions in the samples the mean is
     performed. It accept either a single string or a list of the string with the
     sample names corresponding to the directions along which the mean is performed.
     A single string is equivalent to a list with a single element:
-    ``samples_names = "center"`` is the same as ``samples_names = ["center"]``.
+    ``sample_names = "center"`` is the same as ``sample_names = ["center"]``.
 
-    For an usage example see the doc for ``sum_over_samples``.
+    For a general discussion of reduction operations and a usage example see the
+    doc for :py:func:`sum_over_samples()`.
 
-    The gradient is implemented as follow:
+    The gradient is implemented as follows:
 
     .. math::
 
         \nabla[Std(X)] = 0.5(\nabla[Var(X)])/Std(X)
         = (E[X \nabla X] - E[X]E[\nabla X])/Std(X)
 
     :param tensor: input :py:class:`TensorMap`
-    :param samples_names: names of samples to perform the standart deviation over
+    :param sample_names: names of samples to perform the standart deviation over
     """
     return _reduce_over_samples(
-        tensor=tensor, samples_names=samples_names, reduction="std"
+        tensor=tensor, sample_names=sample_names, reduction="std"
+    )
+
+
+def var_over_samples_block(
+    block: TensorBlock, sample_names: Union[List[str], str]
+) -> TensorBlock:
+    """Computes the variance for a :py:class:`TensorBlock`,
+    combining the samples according to ``sample_names``.
+
+    See also :py:func:`sum_over_samples_block` and :py:func:`std_over_samples`
+
+    :param block:
+        input :py:class:`TensorBlock`
+    :param sample_names:
+        names of samples to compute the variance for
+
+    :returns:
+        a :py:class:`TensorBlock` containing the reduced values and sample labels
+    """
+
+    return _reduce_over_samples_block(
+        block=block, sample_names=sample_names, reduction="var"
     )
 
 
-def variance_over_samples(tensor: TensorMap, samples_names: List[str]) -> TensorMap:
-    r"""Compute the variance of a :py:class:`TensorMap`, grouping samples according to
-    ``samples_names``.
+def var_over_samples(tensor: TensorMap, sample_names: List[str]) -> TensorMap:
```

Comment:
Same here, `var` => `variance`

Reply:
Same as above. I think that the closer we stay to numpy syntax, the more natural it will feel to users.